app/views/alchemy/_menubar.html.erb

```diff
@@ -112,22 +112,24 @@
     </template>
 
     <script type="module">
```

Comment:
Can we try to use [Turbo data-attributes](https://turbo.hotwired.dev/reference/attributes#data-attributes) instead?

```suggestion
    <script type="module" data-turbo-eval="false">
```